packages/create-contentful-app/template/README.md

```diff
@@ -0,0 +1,62 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `yarn configure`
+
```

Comment:
In the cli output we are talking about using `npm` and running `npm run configure`. However, here we are talking about `yarn`. Should be stick to one or mention both at the same?

Reply:
I'd stick to npm. You'll get npm with node, yarn you need to configure. We need to assume any level of experience, installing/configuring yarn may be problematic or require support for some people

Reply:
Corrected the yarn -> npm but will leave the rest of README for documentation.